modules/apps/document-library/document-library-repository-cmis-impl/src/main/java/com/liferay/document/library/repository/cmis/internal/model/CMISFileEntry.java

```diff
@@ -194,6 +194,11 @@ public Date getCreateDate() {
 		return creationDate.getTime();
 	}
 
+	@Override
+	public Date getDisplayDate() {
+		return null;
```

Comment:
Shouldn't this return the create date? That would be consistent with the upgrade you implemented in a previous PR (where you set displayDate = createDate).

Reply:
mmmm
on this case the displayDate was lastPublishDate and on this clase is always null.

Why do you prefer to be create date?
do you want to change both PR?

Reply:
hi @adolfopa , other thing that can affect here:

Taking into account that schedule fields are not shown on edit mode, I do not see the need.

That are reasons so let be null on this case and do not make it mandatory as on KB.

let me know your feedback :)

Reply:
If this can be optional, this I'm fine with it.